src/sagemaker_sklearn_container/serving.py

```diff
@@ -12,11 +12,13 @@
 # language governing permissions and limitations under the License.
 from __future__ import absolute_import
 
+import importlib
 import logging
 import numpy as np
 from sagemaker_containers.beta.framework import (
```

Comment:
why are we still using the sagemaker_containers beta? I thought the resolution was to consume sagemaker_containers instead.

Reply:
I tried that and it didn't do anything. Maybe I'm missing something, but to me it looks like `beta.env` is just `_env`: https://github.com/aws/sagemaker-containers/blob/master/src/sagemaker_containers/beta/framework/__init__.py#L21

Reply:
hmm did you discuss this with sagemaker-containers (since they believe beta is the root cause of this issue)?